openid-sharedsignals-framework-1_0.md

```diff
@@ -511,6 +643,44 @@ The following are hypothetical examples of SETs that conform to the Shared Signa
 ~~~
 {: #subject-custom-type-ex title="Example: SET Containing an SSF Event with a Proprietary Subject Identifier Format"}
 
+# Event Delivery {#event-delivery}
+This section describes the supported methods of delivering SSF Events. It provides SSF profiling specifications for the {{RFC8935}} and {{RFC8936}} specs.
+
+## Stream Configuration Metadata {#delivery-meta}
+Each delivery method is identified by a URI, specified below by the "method"
+metadata.
+
+### Push Delivery using HTTP
+This section provides SSF profiling specifications for the {{RFC8935}} spec.
+
+method
+
+> "urn:ietf:rfc:8935"
+
+endpoint_url
+
+> The URL where events are pushed through HTTP POST. This is set by the
+  Receiver. If a Receiver is using multiple streams from a single Transmitter
+  and needs to keep the SETs separated, it is RECOMMENDED that the URL for each
+  stream be unique.
+
+authorization_header
+
+> If the endpoint_url requires authorization, the receiver SHOULD provide this authorization header in the stream creation/updation. If present, the Transmitter MUST provide this value with every HTTP request to the `endpoint_url`.
```

Comment:
@appsdesh I have incorporated the language from your PR about the authorization_header here.